src/components/EventDetail/TermsConditions/Component.client.tsx

```
@@ -0,0 +1,85 @@
+
+  return (
+    <section className="py-12">
+      <div className="container mx-auto px-4">
+        <h2 className="text-3xl font-bold mb-8">{t('event.termsAndConditions')}</h2>
+
+        <div className="space-y-8">
+          {/* Use event terms if available, otherwise show default terms */}
+          {event.eventTermsAndConditions ? (
+            <div
+              className="whitespace-pre-wrap font-montserrat"
+              dangerouslySetInnerHTML={{ __html: event.eventTermsAndConditions }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)